test/unit/quoteEngine.js

```diff
@@ -113,22 +99,22 @@ describe('Quote Engine tests', () => {
     const [quote1, quote2, quote3] = quoteEngine(store, productId, amount, MIN_COVER_PERIOD, 1);
```

Comment:
i changed the quote amount to `75000` ETH so that pool 1 is skipped and pool 18 (capacity filled) and pool 22 (partially filled).

the resulting values looks good